src/ophyd_async/panda/panda.py

```diff
@@ -116,6 +138,75 @@ def _block_name_number(block_name: str) -> Tuple[str, Optional[int]]:
     return block_name, None
 
 
+def seq_table_from_rows(*rows: SeqTableRow):
+    return seq_table_from_arrays(
+        repeats=np.ndarray([row.repeats for row in rows], dtype=np.uint16),
+        trigger=[row.trigger for row in rows],
+        position=np.ndarray([row.position for row in rows], dtype=np.int32),
+        time1=np.ndarray([row.time1 for row in rows], dtype=np.uint32),
+        outa1=np.ndarray([row.outa1 for row in rows], dtype=np.bool_),
+        outb1=np.ndarray([row.outb1 for row in rows], dtype=np.bool_),
+        outc1=np.ndarray([row.outc1 for row in rows], dtype=np.bool_),
+        outd1=np.ndarray([row.outd1 for row in rows], dtype=np.bool_),
+        oute1=np.ndarray([row.oute1 for row in rows], dtype=np.bool_),
+        outf1=np.ndarray([row.outf1 for row in rows], dtype=np.bool_),
+        time2=np.ndarray([row.time2 for row in rows], dtype=np.uint32),
+        outa2=np.ndarray([row.outa2 for row in rows], dtype=np.bool_),
+        outb2=np.ndarray([row.outb2 for row in rows], dtype=np.bool_),
+        outc2=np.ndarray([row.outc2 for row in rows], dtype=np.bool_),
+        outd2=np.ndarray([row.outd2 for row in rows], dtype=np.bool_),
+        oute2=np.ndarray([row.oute2 for row in rows], dtype=np.bool_),
+        outf2=np.ndarray([row.outf2 for row in rows], dtype=np.bool_),
+    )
+
+
+def seq_table_from_arrays(
```

Comment:
We will use this method later when we need more efficient batch production of tables for the motion stuff...